### DISCUSSION TITLE
Twitter like design for mastodon? (new theme)

### DISCUSSION THREAD

**Discussion by iMagesh** [mastodon/mastodon#25507] [General]
Hi folks,

I just wanted to share an idea to see what people think about it.
Would it be helpful if there was a Twitter-like theme for Mastodon? An independent frontend repo for people to use?

I personally like Twitter's UI a lot (maybe I'm used to it), so I was thinking if Mastodon users had the option to change UI to a Twitter-like theme, a lot of people might like it, right? 

Just thinking out loud. What do you people think?

**Comment by andypiper** [comment]
There are already third party apps and front-ends that provide this, such as [Elk](https://github.com/elk-zone/elk).

**Reply by iMagesh** [reply]
@andypiper Thank you for sharing it, I didn't know. I'll check out Elk.

**Comment by itbeard** [comment]
Already exists https://github.com/ronilaukkarinen/mastodon-bird-ui :)
